### PR TITLE
Hidebutton

### PR DESCRIPTION
**Pull request by Julian-A-Mort**
Primarily a jss update,

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,24 +102,24 @@ <h1 id = "page-title" class = "drop-shadow-2xl text text-center font-Rowdies tex
...
             <button id="history-button" class=" dark:md:hover:bg-yellow-50 p-1.5 bg-orange-300 text-black font-thin">View Historical Results</button>
             <br>
-            <button id="clear-history">clear history</button> <!--used for testing-->
+            <button id="clear-history">clear history</button> 
         </div>
         </section>
     <br>
     
-        <!-- Current Results -->
-        <section>
```
Comment (ajayshans) [inline index.html:111]: @Julian-A-Mort any reason why this section was removed? Looks like we would lose the youtube preview if we do

File: index.html [hunk 2]
```diff
@@ -102,24 +102,24 @@ <h1 id = "page-title" class = "drop-shadow-2xl text text-center font-Rowdies tex
...
+        <br>
+        <button id="retry-button">Try Again!</button>
+    </div>
+
+
+
+
+        <div id="movieDisplay"></div>
+
+        <p id="messageDisplay"></p>
```
Comment (ajayshans) [inline index.html:136]: I think if we replace lines 120-122 here with the red lines 110-122 it should fix the problem

**Review by ajayshans — CHANGES_REQUESTED** [review]
Ensure we retain youtube preview by replacing green lines 120-122 in index.html file with red lines 110-122

**Comment by ajayshans** [comment]
Hi @Julian-A-Mort similar comment to Bea's pull request following the above changes. Once above changes have been made, could you please run the below:
```
git pull origin main
<resolve any conflicts>
```
Final output should look like the codebase in main branch + your new changes and the only js file that should remain is script.js

**Comment by Julian-A-Mort** [comment]
Change made to include youtube display. Please check again...

**Review by ajayshans — APPROVED** [review]

**Review by itsbbea — APPROVED** [review]

**Pull request merged**